package.json

```diff
@@ -17,7 +17,8 @@
     "generate-browsers-json": "./scripts/generate-browsers-json.js",
     "clean": "rm -rf public/packs/*",
     "prebuild": "yarn run clean",
-    "build": "webpack && yarn generate-browsers-json",
+    "build": "yarn build:js",
+    "build:js": "concurrently 'yarn:webpack' 'make browsers.json'",
```

Comment:
just for my own edification, in other project's I've seen you use `run-p` so we 'd have like:
```suggestion
    "build:js": "run-p build:js:*",
    "build:js:webpack": "yarn:webpack",
    "build:js:browsers-json": "make browsers.json",
```
is there a reason we didn't do that here? to me seems like adding a new dependency either way so idk if one is better/preferred/more common than the other

Reply:
I think I recall being faced with this choice with some recent transitive dependency having a vulnerability, and discovering that [`npm-run-all`](https://www.npmjs.com/package/npm-run-all) (`run-p`) hasn't received a new release in 5 years, which was a motivator to find a maintained alternative. `concurrently` also makes it a little easier to work with non-NPM commands like what's used here.

But ultimately, yes, they're largely used to the same end, and I think your proposed alternative could work just as well.